lib/ci-helper.ts

```diff
@@ -183,24 +185,43 @@ export class CIHelper {
                 meta.commitInGitGit = undefined;
                 result = true;
             }
+            bases.add(`-p${info.baseCommit}`);
+            heads.add(`-p${info.headCommit}`);
+        }
+
+        if (heads.size > 0) {
+            /*
+             * Generate throw-away octopus merges to combine multiple commit
+             * ranges into a single one.
+             */
+            const octopus = async (set: Set<string>): Promise<string> => {
+                const array = Array.from(set);
+                if (array.length === 1) {
+                    return array[0];
+                }
+                return await git(["commit-tree", ...array, emptyTreeName],
+                                 { workDir: this.workDir, stdin: "()" });
```

Comment:
Just curious - why use stdin instead of -m command line parm?

Reply:
Oh shucks. That `-m` option is [right there](https://git-scm.com/docs/git-commit-tree#Documentation/git-commit-tree.txt--mltmessagegt). I thought I remembered that the `git commit-tree` command did _not_ accept `-m`... Will fix.

Reply:
Fixed.